components/org.apache.stratos.manager/src/main/database/SM.sql

```diff
@@ -0,0 +1,79 @@
+SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0;
+SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0;
+SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='TRADITIONAL,ALLOW_INVALID_DATES';
+
+CREATE SCHEMA IF NOT EXISTS `StratosManager` DEFAULT CHARACTER SET utf8 COLLATE utf8_general_ci ;
+USE `StratosManager` ;
+
+-- -----------------------------------------------------
+-- Table `StratosManager`.`Cluster`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `StratosManager`.`Cluster` (
+  `clusterId` VARCHAR(255) NULL,
```

Comment:
@JaneAarthy , why this clusterId, which is also declared as the Primary key,  is defined as a 'NULL' ?